closure/goog/soy/soy_testhelper.js

```diff
@@ -64,44 +72,79 @@ function makeSanitizedContent(content, kind) {
 var example = {};
 
 
-example.textNodeTemplate = function(opt_data, opt_sb, opt_injectedData) {
-  assertNotNull(opt_data);
-  assertNotUndefined(opt_data);
-  return goog.string.htmlEscape(opt_data.name);
+/**
+ * @param {{name: string}} data
+ * @param {null=} opt_sb
```

Comment:
I think it is a legacy [parameter or something.. I can't figure out its full name.